prediction/src/preprocess/load_ct.py

```diff
@@ -127,14 +127,26 @@ def extract_spacing_dcm(self):
         # Every DICOM file have the same PixelSpacing
         spacing = self.meta[0].PixelSpacing
         # Taking into account ijk -> xyz transformation
-        return np.asarray([spacing[1], spacing[0], slice_thickness])
+        if self.xyz_order:
+            return [spacing[1], spacing[0], slice_thickness]
+        else:
+            return [slice_thickness, spacing[0], spacing[1]]
 
     def extract_spacing_mhd(self):
-        # Again, taking into account ijk -> xyz transformation
-        return list(reversed(self.meta.GetSpacing()))
+        if self.xyz_order:
+            return list(reversed(self.meta.GetSpacing()))
+        else:
```

Comment:
No need for an else; you've just called ``return`` on the previous lin. (and elsewhere)